CSFDVirtualKeyBoard.py

```diff
@@ -67,6 +73,17 @@ def VirtualKeyBoardEntryComponent(keys, selectedKey, shiftMode=False):
 		key_shift = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_shift_fhd.png'))
 		key_shift_sel = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_shift_sel_fhd.png'))
 		key_space = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_space_fhd.png'))
+	else:
+		h = 87
+		key_backspace = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_backspace_fhd.png'))
+		key_bg = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_bg_fhd.png'))
+		key_clr = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_clr_fhd.png'))
+		key_esc = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_esc_fhd.png'))
+		key_ok = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_ok_fhd.png'))
+		key_sel = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_sel_fhd.png'))
+		key_shift = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_shift_fhd.png'))
+		key_shift_sel = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_shift_sel_fhd.png'))
+		key_space = LoadPixmap(cached=True, path=resolveFilename(SCOPE_CURRENT_SKIN, '/usr/lib/enigma2/python/Plugins/Extensions/CSFD/icons/vkey_space_fhd.png'))
```

Comment:
Nemali by tieto súbory mať postfix _wqhd namiesto _fhd? Lebo v ďalšom commite pridávaš nové práve _wqhd png, no nikde nevidím ich použitie.

Reply:
Prepáč,áno to som prehliadol,na tie fhd png som pozabudol a neprepísal na wqhd.png. Preto sa mi zdalo,že medzi okienkami  vo virtuálnej klávesnice sú medzery,ale nedošlo mi to prekontrolovať.

Reply:
Ok, tak to este prosim uprav a mozem to mergnut a vydat von.